src/Protocol/ForgeHandshake.h

```diff
@@ -44,6 +44,8 @@ class cForgeHandshake
 	void HandleModList(cClientHandle * a_Client, const char * a_Data, size_t a_Size);
 	void HandleHandshakeAck(cClientHandle * a_Client, const char * a_Data, size_t a_Size);
 
+	void SetError(const AString & message);
```

Comment:
You should probably doxy-comment what this function is intended to do - does it drop the client connection, does it raise an exception, does it phone Mojang to complain... :)